### ISSUE TITLE
`Content.cid` should be `bytes`

### ISSUE THREAD

**Issue by AuHau** [codex-storage/codex-contracts-eth#107]
https://github.com/codex-storage/codex-contracts-eth/blob/57e8cd5013325f05e16833a5320b575d32a403f3/contracts/Requests.sol#L26

**Comment by markspanbroek** [comment]
Agreed that this should be the CID in binary form, not in string form. But it should `bytes` instead of `bytes32`, because a CID [has a variable length](https://github.com/multiformats/cid#how-does-it-work).
